tests/elfs/reloc.c

```diff
@@ -0,0 +1,11 @@
+/**
+ * @brief a program to test R_BPF_64_64 relocation handling
+ */
+
+typedef unsigned long int uint64_t;
+typedef unsigned char uint8_t;
+
+extern uint64_t entrypoint(const uint8_t *input) {
+  uint64_t (*ptr)(const uint8_t *) = entrypoint;
```

Comment:
Why will this be a `R_BPF_64_64` relocation?
Because of the function pointer?

Reply:
because an imm64 is loaded and the imm64 is a relocatable address